arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'How to generate a keyset for a DAC'
+description: This how-to will help you configure the DAC in your chain.
+author: TheGreatSoshiant
+sme: TheGreatSoshiant
+sidebar_position: 4
+target_audience: 'Developers and node operators deploying and maintaining AnyTrust-based chains.'
+content_type: how-to
+---
+
+import UnderConstructionPartial from '../../partials/\_under-construction-banner-partial.md';
+
+<UnderConstructionPartial />
+
+The DAC has _N_ members; the AnyTrust protocol assumes that a minimum of _H_ _DAC_ members maintain integrity. _H_ is the minimum number of trusted committee members on AnyTrust chains, configurable by the chain's owner via the `assumed-honest` parameter in the keyset. In scenarios where `K = (N + 1) - H` members of the DAC pledge to grant access to specific data, they must sign and attest they have the data for storage to be considered successful.
+
+Each DAC member gets their own set of BLS public and private keys. It's important for every member to create their own new and secure BLS keys. They should do this on their own and make sure these keys are random and only for their use. If you need help generating BLS keys, please refer to our guide on generating keys in the Arbitrum documentation: [Generating BLS Keys](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key).
```

Comment:
The link doesn't seem to go to the right page

Reply:
It should now, pls lmk if the link it points to now is relevant.